quic/s2n-quic-core/src/packet/interceptor.rs

```diff
@@ -44,6 +45,18 @@ pub trait Interceptor: 'static + Send {
         let _ = ack;
     }
 
+    #[inline(always)]
+    fn intercept_rx_local_address(&mut self, subject: &Subject, addr: &mut LocalAddress) {
+        let _ = subject;
+        let _ = addr;
+    }
+
+    #[inline(always)]
+    fn intercept_rx_remote_address(&mut self, subject: &Subject, addr: &mut RemoteAddress) {
+        let _ = subject;
+        let _ = addr;
+    }
+
     #[inline(always)]
     fn intercept_rx_remote_port(&mut self, subject: &Subject, port: &mut u16) {
```

Comment:
do we still need this one?

Reply:
Probably not. I guess I can remove it since it's not stable